lib/format.js

```diff
@@ -12,13 +13,18 @@ export default function format(editor) {
 		if (!report) {
 			return [];
 		}
+
 		const [{messages}] = report.results;
+
 		return messages.map(message => {
+			const [err, range = []] = withErrorHandling(selectMessageRange)(editor, message);
```

Comment:
We should not handle the error or display it nicely. It's a severe edge-case and should cause it to fail.